### PR TITLE
[KOGITO-2563] Add Red Hat headers to all sources in Kogito runtimes

### PR DESCRIPTION
**Pull request by bxf12315**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by bxf12315** [comment]
@cristianonicolai com.mycila.license-maven-plugin upgared to 4.0.rc1,so we can use license.git.copyrightCreationYear. please review it.

**Comment by cristianonicolai** [comment]
@bxf12315 can you link it to a JIRA and try to fix the build error please.

**Comment by bxf12315** [comment]
> @bxf12315 can you link it to a JIRA and try to fix the build error please.

The RP from https://github.com/Cory-jbpm/kogito-runtimes/,not from https://github.com/bxf12315/kogito-runtimes/,so the PR build failed.

**Comment by bxf12315** [comment]
jenkins retest this

**Comment by bxf12315** [comment]
jenkins retest this

**Comment by Ginxo** [comment]
jenkins retest this

**Comment by Ginxo** [comment]
jenkins retest this

**Comment by Ginxo** [comment]
jenkins retest this

**Comment by bxf12315** [comment]
@cristianonicolai I upgraded com.mycila:license-maven-plugin to 4.0.rc1.
The PR has two commits, first just include pom.xml and licensehead.txt.I want to it is easy to review.

File: pom.xml [hunk 1]
```diff
@@ -83,6 +87,51 @@
     </repository>
   </repositories>
 
+  <build>
+    <pluginManagement>
```
Comment (cristianonicolai) [inline pom.xml:91]: please move configuration **kogito-build-parent** pom.xml

File: pom.xml [hunk 2]
```diff
@@ -48,6 +48,10 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- license plugin -->
+    <version.kogito.license.plugin>4.0.rc1</version.kogito.license.plugin>
```
Comment (cristianonicolai) [inline pom.xml:53]:
```suggestion
    <version.license.plugin>4.0.rc2</version.license.plugin>
```

File: pom.xml [hunk 3]
```diff
@@ -48,6 +48,10 @@
...
+    <skipLicensesCheck>true</skipLicensesCheck>
```
Comment (cristianonicolai) [inline pom.xml:54]: should it be false by default so build fails in case of missing headers?

**Review by cristianonicolai — COMMENTED** [review]
@bxf12315 please review conflicting files

**Pull request closed without merging**